.github/workflows/test.yml

```diff
@@ -22,16 +22,64 @@ env:
   MYSQL_HOSTNAME_B: 'mysql-b'
 
 jobs:
+  dependency-graph:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    container: swift:jammy
```

Comment:
Isn't this a tiny bit of inconsistency? 🤔
I think we usually explicitly specify the Swift version.

Reply:
Probably not for the dependency submission graph, it just needs to build it.

That said, I feel like the dependency graph stuff could be integrated directly into the main Vapor test stuff if it isn't already

Reply:
It is. This package just has the "needs extra services and the main CI doesn't have that capability quite yet" problem like most of Fluent's repos.

Reply:
And I stopped explicitly specifying the Swift version for certain ancillary CI jobs (like this one) in the main CI long ago.